cogs5e/models/sheet/spellcasting.py

```diff
@@ -118,7 +118,7 @@ def get_spell(self, spell):
     def cast(self, spell, level):
         """
         Uses the resources to cast *spell* at *level*.
-
+ABILITY
```

Comment:
?

Reply:
I'm not entirely sure either to be entirely honest. I don't remember adding that.